decidim-conferences/app/cells/decidim/conferences/content_blocks/highlighted_conferences_cell.rb

```diff
@@ -3,33 +3,26 @@
 module Decidim
   module Conferences
     module ContentBlocks
-      class HighlightedConferencesCell < Decidim::ViewModel
-        delegate :current_user, to: :controller
+      class HighlightedConferencesCell < Decidim::ContentBlocks::HighlightedParticipatorySpacesCell
+        BLOCK_ID = "highlighted-conferences"
 
-        def show
-          render if highlighted_conferences.any?
-        end
+        delegate :current_user, to: :controller
 
-        def highlighted_conferences
+        def highlighted_spaces
           OrganizationPrioritizedConferences.new(current_organization, current_user)
         end
 
         def i18n_scope
           "decidim.conferences.pages.home.highlighted_conferences"
         end
 
-        def decidim_conferences
-          Decidim::Conferences::Engine.routes.url_helpers
+        def all_path
+          Decidim::Conferences::Engine.routes.url_helpers.conferences_path
         end
 
         private
 
-        def cache_hash
-          hash = []
-          hash.push(I18n.locale)
-          hash.push(highlighted_conferences.map(&:cache_key_with_version))
-          hash.join(Decidim.cache_key_separator)
-        end
+        def block_id = BLOCK_ID
```

Comment:
Same comment as in [highlighted_assemblies_cell.rb](https://github.com/decidim/decidim/pull/10920/files#diff-b3db5c57a4634c36562d7145ffcab03aebd4955fa796a9daa5b031fac9bcdb9e)